### DISCUSSION TITLE
Welcome to AllyGator Discussions!

### DISCUSSION THREAD

**Discussion by kovurusaisachin** [kovurusaisachin/AllyGator#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kovurusaisachin** [comment]
Hi @noopurthanvi  @prashant-kapri @MaitreyiSrinivasan @prashantkapri , I have added the back-end structure

**Reply by MaitreyiSrinivasan** [reply]
Hey @kovurusaisachin. That's cool! I have created the homepage on the front-end. Let me know your thoughts.
